pom.xml

```diff
@@ -37,6 +37,7 @@
         <module>conformance</module>
         <module>standardization</module>
         <module>dataModel</module>
+        <module>examples</module>
```

Comment:
If we are adding it to the main build, we should remove the `examples` profile at the end of the pom file.